loc/translations-import/vscode-powerplatform.cs.xlf

```diff
@@ -1,462 +1,310 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
To keep Main clean for releases should one be needed prior to the new localization files, we'll keep this branch and PR unmerged until @toshio-msft and team have the recycled import XLFs pushed into this branch.

Reply:
Got the import XLFs in, and tested the process with the supplied translations: 
✔️Spot checked Import XLF files
✔️build-time generation of Bundle and Package artifacts works as expected
✔️French strings displayed correctly on local Windows box (minus new strings which haven't been translated)
✔️Japanese and German strings displayed correctly GitHub CodeSpaces web client (minus new strings which haven't been translated)